Makefile

```diff
@@ -59,9 +61,16 @@ formatter:
 	poetry run black rasa tests
 
 lint:
-	poetry run flake8 rasa tests
+    # Ignore docstring errors when running on the entire project
+	poetry run flake8 rasa tests --extend-ignore D
 	poetry run black --check rasa tests
 
+branch ?= master
+lint-docs:
```

Comment:
I would rename this to `lint-docstrings:` because it's more explicit. "docs" can be misinterpreted with the actual documentation (related but not the same thing